Makefile

```diff
@@ -159,9 +158,13 @@ deploy-domain-arm-resources: domains domains-composed-variables arm-deployment #
 
 validate-domain-arm-resources: set-what-if domains domains-composed-variables arm-deployment ## Validate what Azure resources will be deployed. Usage: make validate-domain-arm-resources
 
-domains-infra-init: bin/terrafile domains-composed-variables set-azure-account
-	./bin/terrafile -p terraform/domains/infrastructure/vendor/modules -f terraform/domains/infrastructure/config/zones_Terrafile
+.PHONY: vendor-domain-infra-modules
+vendor-domain-infra-modules:
+	rm -rf terraform/domains/infrastructure/environment_domains/vendor/modules/domains
+	TERRAFORM_MODULES_TAG=stable
+	git clone --depth=1 --single-branch --branch ${TERRAFORM_MODULES_TAG} https://github.com/DFE-Digital/terraform-modules.git terraform/domains/infrastructure/vendor/modules/domains
```

Comment:
Can we write it `git -c advice.detachedHead=false clone...` to suppress the warning message

Reply:
fixed